src/passes/Print.cpp

```diff
@@ -25,6 +25,13 @@
 
 namespace wasm {
 
+static int forceFull() {
```

Comment:
Name could be misinterpreted to mean "forcibly set the `full` flag". Maybe `shouldForceFull`?